lib/widgets/responsive_builder.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:hackathon_flutterturkiye_org/core/utils/responsive_helper.dart';
+
+class ResponsiveBuilder extends StatelessWidget {
+  const ResponsiveBuilder({
+    Key key,
+    @required this.mobile,
+    this.tablet,
+    @required this.desktop,
+  })  : assert(mobile != null, "Mobile cannot be empty."),
+        assert(desktop != null, "Desktop cannot be empty."),
+        super(key: key);
+
+  /// 0px - 799px
+  final Widget mobile;
+
+  /// 800px - 1199px
+  final Widget tablet;
+
+  /// 1200px
+  final Widget desktop;
```

Comment:
Why we have a difference between desktop and tablets. An ipad can be as big as a laptop for example.

Reply:
I tried to use the standards like Bootstrap, Microsoft. Generally, sources use three different devices. [Example](https://docs.microsoft.com/en-us/windows/uwp/design/layout/screen-sizes-and-breakpoints-for-responsive-design)

Reply:
Yes but that is not what we do with Flutter. We care about only the size. So let's try to keep it as small medium large etc.

Reply:
What is your suggestion? Should we remove the helper?

Reply:
We should rename them to something more generic. Because a desktop can be as small as a tablet and a tablet can be as big as a Desktop

Reply:
We can change the names to **Small, Medium, Large**.